fsharp-backend/src/LibBackend/TraceInputs.fs

```diff
@@ -260,6 +247,7 @@ let get404s (limit : Limit) (canvasID : CanvasID) : Task<List<F404>> =
       events
       |> List.filter (fun e ->
         not (List.exists (fun (_tlid, h) -> matchEvent h e) handlers))
+      |> List.map (fun ((s, n, m), ts, v) -> (s, n, m, ts, v))
```

Comment:
I'd much rather these be typed out - these 1-character bindings always leave me re-tracing the code finding context.